pkgs/leak_tracker/README.md

```diff
@@ -1,6 +1,32 @@
 [![pub package](https://img.shields.io/pub/v/leak_tracker.svg)](https://pub.dev/packages/leak_tracker)
 [![package publisher](https://img.shields.io/pub/publisher/leak_tracker.svg)](https://pub.dev/packages/leak_tracker/publisher)
 
-Coming soon!
+Coming soon! See https://github.com/flutter/devtools/issues/3951 and https://github.com/flutter/devtools/issues/5606.
 
-See full documentation at https://github.com/dart-lang/leak_tracker.
+## What is this?
+
+This is a framework for detecting memory issues in Dart and Flutter applications.
+
+It enables:
+
+1. [Memory leak auto-detection](https://github.com/dart-lang/leak_tracker/blob/main/doc/DETECT.md) for applications and tests.
+2. [Memory usage tracking and auto-snapshotting](https://github.com/dart-lang/leak_tracker/blob/main/doc/USAGE.md).
+
+## Usage
+
+### Leak tracking
+
+First, [understand leak tracking concepts](https://github.com/dart-lang/leak_tracker/blob/main/doc/CONCEPTS.md).
+
+TODO(polina-c): add usage information.
```

Comment:
This todo will be what people see when they visit the package on pub.dev - this will be part of the public docs of the package.

Reply:
Yes, I want people to see that this is still under construction and after stabilization this information will be added.